src/emqtt.erl

```diff
@@ -246,7 +246,6 @@
           ack_timer       :: tref() | undefined,
           retry_interval  :: pos_integer(),
           retry_timer     :: tref() | undefined,
-          session_present :: undefined | boolean(),
```

Comment:
Q: It was also implicitly part of `info/1` call result, so in a sense a library user won't be able anymore to tell if a session was present on the broker side. Is it expected?

Reply:
good point.